src/containers/StructurePage/resourceComponents/Resource.tsx

```diff
@@ -145,11 +225,37 @@ const Resource = ({ resource, onDelete, dragHandleProps, currentNodeId }: Props)
     onSettled: () => qc.invalidateQueries(compKey),
   });
 
+  const id = getIdFromUrn(resource?.contentMeta?.contentUri);
+  const { data: article } = useDraft({ id: id! }, { enabled: !!id });
+
+  useEffect(() => {
+    if (article) {
+      const isAproachingRevision = !!getCountApproachingRevision(article);
+      setAproachingRevision(isAproachingRevision);
+    }
+  }, [article]);
+
+  const { data: userData } = useAuth0Users(
+    { uniqueUserIds: article?.responsible?.responsibleId! },
+    { enabled: !!article?.responsible?.responsibleId },
+  );
+
+  useEffect(() => {
+    if (userData?.length) {
+      setResponsible(userData[0].name);
+    }
+  }, [userData]);
+
   const contentType =
     resource.resourceTypes.length > 0
       ? getContentTypeFromResourceTypes(resource.resourceTypes).contentType
       : 'topic-article';
 
+  const contentTypeName =
+    resource.resourceTypes.length > 0
+      ? last(resource.resourceTypes)!.name
```

Comment:
Dette trenger du ikke lodash for. Bare bruk `resource.resourceTypes.length - 1` istedenfor, så slipper vi å gjøre bundle-sizen større enn del allerede er!